pkgs/data/icons/la-capitaine-icon-theme/default.nix

```
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "la-capitaine-icon-theme";
+  version = "0.6.1";
+
+  src = fetchFromGitHub {
+    owner = "keeferrourke";
+    repo = "la-capitaine-icon-theme";
```

```suggestion
    repo = pname;
```

makes sense to reuse it.